packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -8,14 +8,15 @@
 
 %global build_agent (0%{?suse_version} == 0) && ((0%{?fedora} > 28) || (0%{?rhel} > 0))
 %global legacy_agent (0%{?rhel} == 5) || (0%{?rhel} == 6)
+%global build_fact_plugin (0%{?rhel} > 0 && 0%{?rhel} <= 7)
```

Comment:
After reading this, I do wonder about Suse.

Reply:
I'll leave that as an exercise to the reader, errr, someone who knows more about our SUSE bits.

Reply:
We ship our own sub-man for suse: http://koji.katello.org/koji/packageinfo?packageID=1006, so we should be good?

@jturel might know more

Reply:
The subman version we ship for suse doesn't need our workaround - so this is good! :taco: